invokeai/frontend/web/src/features/parameters/types/parameterSchemas.ts

```diff
@@ -126,6 +126,13 @@ const zParameterT2IAdapterModel = zModelIdentifierField;
 export type ParameterT2IAdapterModel = z.infer<typeof zParameterT2IAdapterModel>;
 // #endregion
 
+// #region I2I Initial Image
+const zParameterInitialImage = z.string();
+export type ParameterInitialImage = z.infer<typeof zParameterInitialImage>;
+export const isParameterInitialImage = (val: unknown): val is ParameterInitialImage =>
+  zParameterInitialImage.safeParse(val).success;
+// #endregion
+
```

Comment:
This will cause problems. If you look at the `GenerationState` type, you'll see the type for `initialImage`:
```ts
initialImage?: { imageName: string; width: number; height: number };
```